packages/cli/globalTeardown.js

```diff
@@ -0,0 +1,5 @@
+import { teardown } from 'jest-dev-server'
+
+export default async function globalTeardown() {
+  await teardown()
```

Comment:
Maybe part of the teardown could be to delete the `CONFIG_DIR_PATH` folder created in the setup script?

Reply:
@PaulLeCam , I added the removal of the whole `TEMP_DIR_PATH` whose `CONFIG_DIR_PATH` is a subdirectory: https://github.com/ceramicstudio/js-composedb/pull/2/files#diff-2eee76cf110a51a727cb612325f04406c422d0b3891d1f9bd8478abfafbc94cbR6

Does it work for you :) ?